wagtail/wagtailusers/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     url(r'^$', users.index, name='wagtailusers_index'),
     url(r'^new/$', users.create, name='wagtailusers_create'),
+    url(r'^(\d+)/toggle/$', users.toggle, name='wagtailusers_toggle'),
```

Comment:
Instead of using "toggle" here. Could this view be renamed to "/active/" and made to respond to "PUT" (which sets the active flag to true) and "DELETE" (which clears it) requests. This solves a potential issue where if the state gets out of sync between the frontend and the backend (eg somebody else is toggling the same flag in another session).